src/graphics/resources/buffer.rs

```diff
@@ -49,21 +76,31 @@ fn quad_vertices(uv_offset: Vec2) -> [ImageVertex; 6] {
         canon: vec2(x, y),
         uv: vec2(u, v),
     };
+    let (nu, pu) = (0. - du, 1. + du);
+    let (nv, pv) = (0. - dv, 1. + dv);
```

Comment:
Is it negative and positive?

Reply:
Yes exactly, I'll add a comment since it's not clear right away.